services/ui-src/src/components/reports/StandardReportPage.test.tsx

```diff
@@ -1,82 +1,34 @@
 import { render, screen } from "@testing-library/react";
-import userEvent from "@testing-library/user-event";
 import { axe } from "jest-axe";
 // components
 import { ReportContext, StandardReportPage } from "components";
 // utils
-import { useUser } from "utils";
 import {
-  mockAdminUser,
   mockForm,
   mockReportContext,
   mockStandardReportPageJson,
-  mockStateUser,
   RouterWrappedComponent,
 } from "utils/testing/setupJest";
 
-const mockUseNavigate = jest.fn();
-jest.mock("react-router-dom", () => ({
-  useNavigate: () => mockUseNavigate,
-  useLocation: jest.fn(() => ({
-    pathname: "/mock",
-  })),
-}));
-
-jest.mock("utils/auth/useUser");
-const mockedUseUser = useUser as jest.MockedFunction<typeof useUser>;
-
 const standardPageSectionComponent = (
   <RouterWrappedComponent>
     <ReportContext.Provider value={mockReportContext}>
       <StandardReportPage
         route={{ ...mockStandardReportPageJson, form: mockForm }}
       />
-      <button form={mockForm.id} type="submit">
-        submit
-      </button>
     </ReportContext.Provider>
   </RouterWrappedComponent>
 );
 
 describe("Test StandardReportPage", () => {
-  afterEach(() => jest.clearAllMocks());
-
   test("StandardReportPage view renders", () => {
-    mockedUseUser.mockReturnValue(mockStateUser);
     render(standardPageSectionComponent);
     expect(screen.getByTestId("standard-page")).toBeVisible();
   });
-
-  test("StandardReportPage updates report field data on successful fill from state user", async () => {
-    mockedUseUser.mockReturnValue(mockStateUser);
-    const result = render(standardPageSectionComponent);
-    const form = result.container;
-    const mockField = form.querySelector("[name='mock-text-field']")!;
-    await userEvent.type(mockField, "mock input");
-    const submitButton = form.querySelector("[type='submit']")!;
-    expect(submitButton).toBeVisible();
-    await userEvent.click(submitButton);
-    expect(mockReportContext.updateReport).toHaveBeenCalledTimes(1);
-    expect(mockUseNavigate).toHaveBeenCalledTimes(1);
-  });
```

Comment:
Request: Related to the above comment. We can reuse this test for the navigation test. Just strip out anything related to user type and updating the fields or checking that report context was updated and we should be good to go.

Reply:
yeah this is fine.